protobuf-format.md

```diff
@@ -0,0 +1,134 @@
+# Protocol Buffers Event Format for CloudEvents  - Version 0.1
+
+## Abstract
+
+The Protocol Buffers Format for CloudEvents (CE) defines the encoding
+of CloudEvents in the Protocol Buffers binary format.
+
+## Status of this document
+
+This document is a working draft.
+
+## 1. Introduction
+
+This specification defines how the [Context
+Attributes](spec.md#context-attributes) defined in the CloudEvents
+specification MUST be encoded in the protocol buffer binary
+format. Transcoding to and from other formats (e.g. JSON) is out of
+the scope of this document.
+
+Protocol Buffers are a language-neutral, platform-neutral extensible
+mechanism for serializing structured data. The [Google reference
+implementation of Protocol
+Buffers](https://github.com/protocolbuffers/protobuf) includes support
+for an interface descriptor language (IDL), and this document makes
+use of language level 3 IDL from Protocol Buffers v3.5.0. CloudEvents
+systems using Protocol Buffers are not mandated to use the IDL or any
+particular implementation of Protocol Buffers as long as they produce
+messages which match the binary encoding defined by the IDL.
+
+
+### 1.1. Conformance
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in
+[RFC2119](https://tools.ietf.org/html/rfc2119).
+
+## 2. Protocol Buffers format
+
+Protocol Buffers provide a binary data serialization format which is
+substantially more compact and efficient to parse when compared to XML
+or JSON, along with a variety of language-specific libraries to
+perform automatic serialization and deserialization. The [Protocol
+Buffers specification defines a well-known encoding
+format](https://developers.google.com/protocol-buffers/docs/encoding)
+which is the basis of this specification. Specifications below may be
```

Comment:
s/may/MAY/
What do you mean by "below"? Do you mean later on in the this doc? Which specs are you referring to?

Reply:
Reworded this section. Apologies for the strange wording.